README.md

```diff
@@ -4,9 +4,9 @@
 
 ActiveModel::Serializer brings convention over configuration to your JSON generation.
 
-AMS does this through two components: **serializers** and **adapters**.
-Serializers describe _which_ attributes and relationships should be serialized.
-Adapters describe _how_ attributes and relationships should be serialized.
+AMS does this through tree components: **serializers** and **adapters**.
```

Comment:
Actually it was a TYPO :smile:
It will still be only two components (probably), but I might need to change serializer's name